### PR TITLE
Added highlight for active tab in edit profile page

### PR DESCRIPTION
**Pull request by mhevyk**
Added styles to make tab highlighted when it is active

### Before

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/e20c9958-aa0d-4726-a03d-d15508ec2ea9

### After

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/b683c054-7e27-4b51-933b-17685c17ea14

### CODE REVIEW

**Review by abalanovsky — APPROVED** [review]

File: src/constants/translations/en/edit-tutor-profile.json [hunk 1]
```diff
@@ -6,6 +6,6 @@
     "profile": "Profile",
     "professionalInfo": "Professional info",
     "notifications": "Notifications",
-    "passwordSecurity": "Password & Security"
+    "passwordAndSecurity": "Password & Security"
```
Comment (dmtrth25) [inline src/constants/translations/en/edit-tutor-profile.json:9]: check `passwordSecurityTab` - `Password & Security` is already exist property. 
Maybe to pass some values to the `common` and reuse it ?

**Review by OliaHrytsak — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

File: src/constants/translations/en/edit-tutor-profile.json (hunk 1 above)
Comment (mhevyk) [reply src/constants/translations/en/edit-tutor-profile.json:9]: I will better fix it in #1928

**Review by dmtrth25 — APPROVED** [review]

File: src/components/sidebar-menu/SidebarMenu.tsx [hunk 2]
```diff
@@ -15,25 +15,29 @@ import { styles } from '~/components/sidebar-menu/SidebarMenu.styles'
...
 
-const SidebarMenu: FC<SidebarMenu> = ({ handleClick, tabsData }) => {
+const SidebarMenu: FC<SidebarMenu> = ({ handleClick, tabsData, activeTab }) => {
   const { t } = useTranslation()
 
   const list = Object.keys(tabsData).map((key) => {
     const tabKey = key as TutorProfileTabsEnum
     const item = tabsData[tabKey]
+
+    const isActiveTab = item.title.endsWith(activeTab)
```
Comment (byavdoshniak) [inline src/components/sidebar-menu/SidebarMenu.tsx:28]: Consider using some `activeTab` id or other identifier instead of checking tittle

**Review by byavdoshniak — APPROVED** [review]

File: src/components/sidebar-menu/SidebarMenu.tsx (hunk 2 above)
Comment (mhevyk) [reply src/components/sidebar-menu/SidebarMenu.tsx:28]: Discussed this solution on the meeting

**Pull request merged**